service-worker/windowclient-navigate/index.html

```diff
@@ -0,0 +1,31 @@
+---
+feature_name: "Service Worker: WindowClient.navigate()"
+chrome_version: 49
+feature_id: 5314750808326144
+---
+
+<h3>Background</h3>
+<p>
+  <code>WindowClient.navigate()</code> allows a service worker to cause a web
```

Comment:
@jpmedley It might be good to document <code>WindowClient.navigate()</code> on MDN and link from there.  WDYT?